Solutions/Marain.Workflow.Storage.Sql.Database/WorkflowInstanceInterest.sql

```diff
@@ -0,0 +1,7 @@
+﻿CREATE TABLE [dbo].[WorkflowInstanceInterest]
+(
+	[Id] INT NOT NULL , 
```

Comment:
I think we should rename this column to `WorkflowInstanceId`

Reply:
I had a long think about this, and ended . WorkflowInstanceId means "stringy thing". I do however think it implies "Id of this thing" so I propose changing it to `WorkflowInstance`